### PR TITLE
Adds education page with videos related to voting and elections.

### PR DESCRIPTION
**Pull request by tjyoung99**

### CODE REVIEW

File: project/src/main/webapp/voting_edu.html
Comment (KathyFeiyang) [inline project/src/main/webapp/voting_edu.html]: Change the name and content description to those of the education page.

File: project/src/main/webapp/voting_edu.html
Comment (KathyFeiyang) [inline project/src/main/webapp/voting_edu.html]: Change the name to "Voter Central Education Page"

File: project/src/main/webapp/voting_edu.html
Comment (KathyFeiyang) [inline project/src/main/webapp/voting_edu.html]: Break this long line into multiple lines. Indent +4 spaces at all wrapped lines.
The same applies for the two iframes below as well.

File: project/src/main/webapp/voting_edu.html
Comment (KathyFeiyang) [inline project/src/main/webapp/voting_edu.html]: nit: "congressional"

**Review by KathyFeiyang — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/voting_edu.html
Comment (suparnabarua) [inline project/src/main/webapp/voting_edu.html]: Nit. Change this to "How to Vote"

File: project/src/main/webapp/voting_edu.html
Comment (suparnabarua) [inline project/src/main/webapp/voting_edu.html]: nit. Change to "Primary Elections Explained"

**Review by suparnabarua — CHANGES_REQUESTED** [review]

**Review by ganewman — APPROVED** [review]
LGTM pending comments from Kathy and Suparna!

File: project/src/main/webapp/voting_edu.html
Comment (suparnabarua) [inline project/src/main/webapp/voting_edu.html]: Can you keep the iframe formatting consistent? e.g. here src is in a new line, in the next ones it's not.

**Review by suparnabarua — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/voting_edu.html
Comment (suparnabarua) [inline project/src/main/webapp/voting_edu.html]: nit. Make the second line "information about..." line up with the first line.

File: project/src/main/webapp/voting_edu.html
Comment (suparnabarua) [inline project/src/main/webapp/voting_edu.html]: nit. The closing iframe tags should line up with the opening tags.

**Review by suparnabarua — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/voting_edu.html
Comment (KathyFeiyang) [inline project/src/main/webapp/voting_edu.html]: image
The iframe video doesn't seem to play. Could you check whether this is also occurring on your end?

File: project/src/main/webapp/voting_edu.html
Comment (KathyFeiyang) [inline project/src/main/webapp/voting_edu.html]: nit: It might not be necessary to mention "voting" in the HTML file name. Perhaps change the name to `education.html`, which is more similar to the other files. However this is totally your call! Feel free to ignore this.

**Review by KathyFeiyang — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/education.html
Comment (kellyktan) [inline project/src/main/webapp/education.html]: nit: remove whitespace at the end of lines.

**Review by kellyktan — CHANGES_REQUESTED** [review]

**Review by KathyFeiyang — APPROVED** [review]
LGTM! Thank you so much for making these changes 👍

File: project/src/main/webapp/education.html
Comment (suparnabarua) [inline project/src/main/webapp/education.html]: nit. Indent this https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

**Review by suparnabarua — APPROVED** [review]
LGTM pending resolution of existing comments

**Comment by kellyktan** [comment]
You could point the hardcoded "Resource" navigation link to your education page.  @ganewman what do you think?

**Comment by ganewman** [comment]
Sounds good! That's what that link is meant for. Feel free to change the name of the link if you want though. The text itself is mostly just a placeholder.

**Comment by tjyoung99** [comment]
Would it be fitting to name the link "Education" , as is, or change the name to be "Voter Education"?

File: project/src/main/webapp/navigation.js
Comment (kellyktan) [inline project/src/main/webapp/navigation.js]: I think this needs a .html?  Getting a 404 when run locally.

**Review by kellyktan — CHANGES_REQUESTED** [review]

**Review by kellyktan — APPROVED** [review]

**Pull request merged**
